components/shooter.py

```diff
@@ -95,8 +92,12 @@ def set_range(self, dist: float) -> None:
             # clamp the range between our minimum and maximum
             dist = min(self.ranges[-1], max(dist, self.ranges[0]))
             self.in_range = False
-        self.centre_rpm = interp(dist, self.ranges, self.centre_rpms)
-        self.outer_rpm = interp(dist, self.ranges, self.outer_rpms)
+        self.centre_target = (
+            interp(dist, self.ranges, self.centre_rpms) * self.RPM_TO_CTRE_UNITS
+        )
+        self.outer_target = (
+            interp(dist, self.ranges, self.outer_rpms) * self.RPM_TO_CTRE_UNITS
+        )
```

Comment:
I think we should keep these as sensible units and convert at the points we communicate with the Talons.